src/natcap/invest/hra.py

```diff
@@ -1703,12 +1703,32 @@ def _parse_criteria_table(criteria_table_path, target_composite_csv_path):
                              engine='python')
     table = df.to_numpy()
 
+    # clean up any leading or trailing whitespace.
+    for row_num in range(table.shape[0]):
+        for col_num in range(table.shape[1]):
+            value = table[row_num][col_num]
+            if isinstance(value, str):
+                table[row_num][col_num] = value.strip()
+
+    # Some fields are required and will lead to cryptic errors if not found or
+    # slightly misspelled.
+    habitat_name_header = 'HABITAT NAME'
+    overlap_section_header = 'HABITAT STRESSOR OVERLAP PROPERTIES'
+    habitat_resilience_header = 'HABITAT RESILIENCE ATTRIBUTES'
+    required_section_headers = {
+        habitat_name_header, overlap_section_header,
+        habitat_resilience_header}
+    missing_sections = required_section_headers - set(table[:, 0])
+    if missing_sections:
+        raise AssertionError(
+            "The criteria table is missing these section headers: "
+            f"{', '.join(required_section_headers)}")
```

Comment:
Does it make sense to print `missing_sections` instead of `required_section_headers` in this message? Or else maybe rephrase to "The criteria table requires the following section headers:"

Reply:
Whoops, thanks for catching this!  I've patched it in 7fff675